src/nplinker/genomics/bigscape/runbigscape.py

```
@@ -15,22 +15,46 @@ def run_bigscape(
     antismash_path: str | PathLike,
     output_path: str | PathLike,
     extra_params: str,
+    version: int = 1,
 ):
-    bigscape_py_path = "bigscape.py"
+    """Runs BiG-SCAPE to cluster BGCs.
+
+    The behavior of this function is slightly different depending on the version of
+    BiG-SCAPE that is set to run using the configuration file.
+    Mostly this means a different set of parameters is used between the two versions.
+    """
+    # switch to correct version of BiG-SCAPE
+    if version == 1:
+        bigscape_py_path = "bigscape.py"
+    elif version == 2:
+        bigscape_py_path = "bigscape-v2.py"
+
     logger.info(
         f'run_bigscape: input="{antismash_path}", output="{output_path}", extra_params={extra_params}"'
     )
 
     try:
         subprocess.run([bigscape_py_path, "-h"], capture_output=True, check=True)
     except Exception as e:
-        raise Exception(f"Failed to find/run bigscape.py (path={bigscape_py_path}, err={e})") from e
+        raise Exception(
```

We should raise a specific exception here, e.g. `FileNotFoundError`